src/lib/y2partitioner/actions/edit_blk_device.rb

```diff
@@ -139,12 +140,29 @@ def used_device_error
         )
       end
 
+      # Error when trying to edit a device that is part of a multidevice filesystem
+      #
+      # @return [String, nil] nil if the device is not part of a multidevice filesystem
+      def multidevice_fs_error
+        fs = device.filesystem
+        return nil unless fs&.multidevice?
+
+        format(
+          # TRANSLATORS: %{name} is replaced by a device name (e.g., /dev/sda1).
+          # Since device names can be rather long, make sure the lines
+          # containing %{name} are sorter than the others.
+          _("The device %{name} belongs to a multi-device filesystem.\n" \
+            "It cannot be edited.\n\n"),
+          name: device.name
+        )
+      end
+
       # Error when trying to edit a device that is part of a multi-device Btrfs
       #
       # @return [String, nil] nil if the device is not part of a Btrfs
       def btrfs_error
         fs = device.filesystem
-        return nil unless fs&.multidevice?
+        return nil unless fs&.type&.is?(:btrfs)
```

Comment:
Nice catch, thank you so much!

Reply:
@joseivanlopez 

Anyway, I'm not sure about using the same "{btrfs|multidevice_fs}_error" method for both since at this time only the Btrfs multi-device filesystem could be edited

> _To modify the settings of the Btrfs, **edit the filesystem** itself instead of ..._

I was trying a few different ways splitting the message and building it conditionally, but none of them liked to me. Let's discuss it in a chat.

Reply:
As discussed, we'll keep those different warnings by now since the Btrfs multi-device scenario really behaves different that the other ones under the hood. And we'll work in the possibility to edit the other multi-device filesystem in a near future.

The other issue

> Umm, so I would get this error when editing the device of a single-device Btrfs? 

has been fixed.